cachi2/core/package_managers/rpm/main.py

```diff
@@ -195,6 +195,16 @@ def _download(lockfile: RedhatRpmsLock, output_dir: Path) -> dict[Path, Any]:
             }
             Path.mkdir(dest.parent, parents=True, exist_ok=True)
 
+        for md in arch.metadata:
```

Comment:
Could you please also update the docstring to include an explanation of this new change in behavior?